src/util/jointjs_util.js

```diff
@@ -0,0 +1,98 @@
+import joint from 'jointjs'
+
+export const createPaper = (element, graph, component) => {
+  // Define paper
+  const paper = new joint.dia.Paper({
+    el: element,
+    gridSize: 1,
+    model: graph
+  })
+
+  // When clicking a JointJS element, emit event containing element model
+  paper.on('cell:pointerdown',
+    (cellView) => {
+      component.$emit('send-element', cellView.model)
+    },
+  )
+}
+
+export const createGraph = () => {
+  // Initialize graph
+  const graph = new joint.dia.Graph()
+
+  // Handle bounding child elements inside parent element
+  graph.on('change:position', (cell, newPosition) => {
+    const parentId = cell.get('parent')
+
+    // Move the element if it doesn't have a parent
+    if (!parentId) return
+
+    // Set originalPosition to the relative position of the column
+    if (!cell.get('originalPosition')) {
+      cell.set('position', cell.previous('position'))
+      cell.set('originalPosition', cell.position({ parentRelative: true }))
+      cell.set('position', newPosition)
+    }
+
+    // Calculate relative offsets
+    let relativePos = cell.position({ parentRelative: true })
+    let offsetY = cell.position({ parentRelative: true }).y - cell.get('originalPosition').y
```

Comment:
Could refactor this by using relativePos in line 39 instead of another cell.position({ parentRelative: true }).